BeRich/ListScreen/Stock.swift

```diff
@@ -43,3 +43,15 @@ public struct Stock: Identifiable {
         self.lowPrice = lowPrice
     }
 }
+
+private let minPriceDefaultValue = 0.0
+private let maxPriceDefaultValue = 1000.0
+extension Stock {
+    static func stockArrayMinPriceValue(_ stocks: [Stock]) -> Double {
```

Comment:
Возможно так будет поисмпатичнее:
```suggestion
    static func stocksMinPriceValue(_ stocks: [Stock]) -> Double {
```

Reply:
Исправил